configure.ac

```diff
@@ -423,6 +421,7 @@ AS_IF([test -n "$PKG_CONFIG"], [
 				AS_IF([test "x$ac_cv_lame$ac_cv_mpg123" = "xyesyes"], [
 						enable_mpeg="yes"
 						HAVE_MPEG=1
+						EXTERNAL_MPEG_REQUIRE="libmpg123"
```

Comment:
@evpobr had to fix the variable, and used `EXTERNAL_MPEG_REQUIRE` to be more consistent with the rest.